### PR TITLE
Add support for COUNT subqueries

### PR DESCRIPTION
**Pull request by angrykoala**
These behave similarly to `EXISTS` subqueries and are defined at https://neo4j.com/docs/cypher-manual/current/syntax/expressions/#count-subqueries

### CODE REVIEW

File: src/Cypher.ts [hunk 1]
```diff
@@ -45,7 +45,8 @@ export { Path, NamedPath } from "./references/Path";
 export { PropertyRef as Property } from "./references/PropertyRef";
 
 // Expressions
-export { Exists } from "./expressions/Exists";
+export { Exists } from "./expressions/subquery/Exists";
+export { Count } from "./expressions/subquery/Count";
```
Comment (MacondoExpress) [inline src/Cypher.ts:49]: It's also exported `count` (the aggregation function), are we happy with that?

**Review by MacondoExpress — APPROVED** [review]
LGTM, there is only the thing about the `count` and `Count` exports in the same namespace, your call!

File: src/Cypher.ts (hunk 1 above)
Comment (angrykoala) [reply src/Cypher.ts:49]: Yeah, that is tricky indeed.

Luckily our model of clauses -> classes and functions -> functions makes a clear distinction of these two. But it is still confusing. Not sure we can do anything about it, as these are also confusing in Cypher itself :sweat_smile:

Comment (MacondoExpress) [reply src/Cypher.ts:49]: Not really, alternative we could go with:
- Using different namespaces.
- Change the export name.

But I would have gone with this solution as well, until works!

**Pull request merged**